data/Pakistan/Assembly/ep-popolo-v1.0.json

```diff
@@ -16719,9 +16719,13 @@
           "url": "https://en.wikipedia.org/wiki/Shazia_Ashfaq_Mattu"
         }
       ],
-      "name": "Shazia Ashfaq Mattu",
+      "name": "Shazia Farid",
       "other_names": [
         {
+          "name": "Shazia Ashfaq Mattu",
+          "note": "alternate"
+        },
+        {
```

Comment:
@tmtmtmtm This seems like a slightly odd change, but I'm guessing it's because she got married, and therefore this is correct? Looking at http://na.gov.pk/en/profile.php?uid=958 it seems like a plausible change.